F1WM/Mapping/CalendarMappingProfile.cs

```diff
@@ -13,9 +13,9 @@ public CalendarMappingProfile()
 			CreateMap<Race, CalendarRace>()
 				.ForMember(api => api.TranslatedName, o => o.MapFrom(db => db.Country.GenitiveName.GetGrandPrixName()))
 				.ForMember(api => api.WinnerRaceResult, o => o.MapFrom(db => db.Results))
-				.ForMember(api => api.FastestLapResult, o => o.MapFrom(db => db.FastestLap));
+				.ForMember(api => api.FastestLapResult, o => o.MapFrom(db => db.FastestLaps));
```

Comment:
Mapping from a collection to a single entity doesn't look good. Please make it the same as in RacesMappingProfile